build.gradle

```diff
@@ -50,8 +50,6 @@ dependencies {
 	// WebSocket Client for Twitch PubSub
 	compile group: 'com.neovisionaries', 			name: 'nv-websocket-client',	version: '2.3'
 
-	// IRC Client [https://github.com/KittehOrg/KittehIRCClientLib]
-	compile group: 'org.kitteh.irc', 				name: 'client-lib', 			version: '3.2.0'
 	// - Transitive Dependency Conflict: Ratpack <-> Kitteh IRC Client
```

Comment:
Nice, if KittehIRC is gone we can remove the entire `Transitive Dependency Conflict: Ratpack <-> Kitteh IRC Client` part (netty dependencies) from the build.gradle.